### PR TITLE
Proxies LinearProgressBar

### PR DESCRIPTION
**Pull request by aaronvine**
### What changed

...

### Why it changed

...

---

- [ ] Change is tested
- [ ] Change is documented
- [ ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

File: testkit/index.js [hunk 1]
```diff
@@ -210,5 +210,6 @@ export {
   toggleSwitchTestkitFactory,
   textTestkitFactory,
   labelTestkitFactory,
-  floatingHelperTestkitFactory
+  floatingHelperTestkitFactory,
```
Comment (lbelinsk) [inline testkit/index.js]: Also needs to be proxied from 
`testkit/enzyme.js` and `testkit/protractor.js`

File: src/LinearProgressBar/LinearProgressBar.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import {LinearProgressBar} from 'wix-ui-backoffice/LinearProgressBar';
```
Comment (lbelinsk) [inline src/LinearProgressBar/LinearProgressBar.js]: you should use the `createHOC` from the wix-ui-core.
Then expose
export default createHOC(LinearProgressBar)

This is in order to support dataHook

Comment (aaronvine) [reply src/LinearProgressBar/LinearProgressBar.js]: added it here https://github.com/wix/wix-ui-backoffice/pull/171

**Comment by bchmn** [comment]
@milkyfruit it had a dark background but it wasn't on purpose - fixed.

**Pull request merged**